Packs/ApiModules/Scripts/CoreIRApiModule/CoreIRApiModule.py

```diff
@@ -3339,3 +3458,114 @@ def get_dynamic_analysis_command(client: CoreClient, args: Dict) -> CommandResul
         outputs=filtered_alerts,
         raw_response=raw_response,
     )
+
+
+def add_tag_to_endpoints_command(client: CoreClient, args: Dict):
+    endpoint_ids = argToList(args.get('endpoint_ids', []))
+    tag = args.get('tag')
+
+    if any(  # check if any filter arguments provided
+        key in args for key in
+        (
+            'endpoint_id_list', 'dist_name', 'ip_list', 'group_name', 'platform', 'alias_name',
+            'isolate', 'hostname', 'status', 'first_seen_gte', 'first_seen_lte', 'last_seen_gte', 'last_seen_lte'
+        )
+    ):
+        endpoint_id_list = argToList(args.get('endpoint_id_list'))
+        dist_name = argToList(args.get('dist_name'))
+        ip_list = argToList(args.get('ip_list'))
+        group_name = argToList(args.get('group_name'))
+        platform = argToList(args.get('platform'))
+        alias_name = argToList(args.get('alias_name'))
+        isolate = args.get('isolate')
+        hostname = argToList(args.get('hostname'))
+        status = args.get('status')
+
+        first_seen_gte = arg_to_timestamp(
+            arg=args.get('first_seen_gte'),
+            arg_name='first_seen_gte'
+        )
+
+        first_seen_lte = arg_to_timestamp(
```

Comment:
```
    if arg is None:
        if required is True:
            raise ValueError(f'Missing "{arg_name}"')
        return None
```

looking at the code of `arg_to_timestamp` looks like if the argument is `None`, it will return `None` if required is `False` which it is by default.